### DISCUSSION TITLE
I can't see all-day tasks in Google Calendar

### DISCUSSION THREAD

**Discussion by Mekann2904** [ivan-lednev/obsidian-day-planner#575] [Q&A]
This is a translation of the following text. 

This is a question about remote calendars.

I would like to know how to display all-day tasks.

Environment: MacBook (M2)
Sync target: Google Calendar
Status: No errors in developer tools

It seems that only all-day tasks are not displayed.Tasks with specified times (start time, end time) are displayed.

Thank you in advance.

**Comment by mister19993** [comment]
Same problem
